modules/nw-modifying-operator-install-config.adoc

```diff
@@ -50,20 +45,34 @@ endif::ignition-config[]
 +
 [source,terminal]
 ----
-$ ./openshift-install create manifests --dir=<installation_directory> <1>
+$ ./openshift-install create manifests --dir=<installation_directory>
 ----
-<1> For `<installation_directory>`, specify the name of the directory that
-contains the `install-config.yaml` file for your cluster.
++
+--
+where:
+
+`<installation_directory>`:: Specifies the name of the directory that contains the `install-config.yaml` file for your cluster.
+--
 
-. Create a file that is named `cluster-network-03-config.yml` in the
-`<installation_directory>/manifests/` directory:
+. Create a stub manifest file for the advanced network configuration that is named `cluster-network-03-config.yml` in the `<installation_directory>/manifests/` directory:
```

Comment:
Not sure there's any reason to create the stub file given that the next step is to open this file in an editor, and then you specify the complete contents of what should be in the file, so there's no need to have created the stub file first; you can just skip this and say to create the file in an editor in the next step.

Reply:
@danwinship, I guess the intent was, the preamble is always required, but I can transfer that into the example YAML files instead and skip this step.

Reply:
Realized why I included this; It's easier to refer to this file as the editor target in the next step rather than trying to stuff this much context into a single step.